app/controllers/datastreams_controller.rb

```
@@ -76,22 +71,9 @@ def self.endpoint_for_datastream(datastream)
 
   private
 
-  LEGACY_API = %w[
-    administrative
-    descriptive
-    content
-    geo
-    identity
-    provenance
-    relationships
-    rights
-    technical
-    version
-  ].freeze
-
   def store_xml(druid:, datastream:, content:)
     endpoint = self.class.endpoint_for_datastream datastream
-    return update_directly(druid: druid, datastream: datastream, content: content) unless LEGACY_API.include? endpoint
+    # return update_directly(druid: druid, datastream: datastream, content: content) unless LEGACY_API.include? endpoint
```

I think we can raise an error here if endpoint is not in LEGACY_API.